### DISCUSSION TITLE
我们到底学到了什么？ | /cn/2005/01/what-did-we-learn/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1572] [Comments]
考完了这门课，大多数都来自作业题，要不就是完全没问题，要不就是完全不会做，感觉很不爽，这门课确实感觉也挺难，不出做过的题大家肯定会挂一大片，没办法。到底是怪老师没有讲清楚知识呢？还是怪我们自己没有好好学？还是怪这么课本身就不适合在我们学院开？其实看看书的后面部分也可以看出随机过程在金融市场上应用较为广泛，还是挺有用、挺有意思的，但是我们的课程根本不可能讲到那里去，课时太少，由此我又想到本科阶段到底 &hellip;

https://yihui.org/cn/2005/01/what-did-we-learn/

**Comment by think-crow** [comment]
益辉大哥！今天晚上突然就想到咱们和美国差距的问题！

- 2010年的自我认知，美国领先中国20年
- 2020年的认知，美国领先中国10年

现今看着Gpt的回答：现在的中国相当于二十世纪中后期的美国。联想到国内的宣传手法，及信息封锁，对这些又有些不确定了。

若只谈科技、教育、医疗及社会制度，不谈文化底蕴，结果会怎样呢？想向您求证一下！

**Reply by yihui** [reply]
这种宏观的问题我无法回答，我无法从高层判断，只能小谈一下个人的生活感受。日常生活的科技中国甩了美国一百条街。教育在美国是中小学松、大学紧，中国是反过来，孰好孰坏我不知道；我对中国的教育内卷极其极其反感，我倾向于少给学生一些压力和灌输，多一些自主和自由。医疗中国甩了美国一千多条街，在美国只能祈祷不要生病，因为看病太慢太难，小病自己熬、不用看，大病看不了。社会制度嘛，反正两家都不怎么好，但我感觉中国有希望变得更好，而美国有变坏的趋势（主要是观念分裂）；目前还是美国更好一些，个人权利有更可靠的保障。

**Reply by think-crow** [reply]
嗯呐，感谢老大哥的回答，非常有帮助！

还有，感谢那两个点！
真细节！和自述一模一样：我一般不会告诉你，我会直接改掉。哈哈哈哈！

**Reply by yihui** [reply]
不能容忍 Markdown 语法不对，列表项的减号后面必须有空格。

相比之下，缺两个标点、多一个换行这两个问题我就放过了。

**Reply by think-crow** [reply]
嘿嘿，好滴，收到！
